utils/constants.go

```diff
@@ -31,7 +31,8 @@ var (
 		Slots: kong.Int(defaultSlots),
 		Healthchecks: &kong.Healthcheck{
 			Active: &kong.ActiveHealthcheck{
-				Concurrency: kong.Int(defaultConcurrency),
+				Concurrency:            kong.Int(defaultConcurrency),
+				HTTPSVerifyCertificate: kong.Bool(true),
```

Comment:
The problem here is that if you set the default to  `true`, there is absolutely no way to set it to `false` even if someone want's to do it.

Reply:
If you do not set the parameter httpsverifycertificate, Kong will default to true?
According to my observation, Kong sets httpsverifycertificate = true by default

Reply:
Yes, the default is true in Kong: https://docs.konghq.com/2.2.x/admin-api/#upstream-object